greynoise/gnql_query.py

```diff
@@ -0,0 +1,30 @@
+from connectors.core.connector import ConnectorError, get_logger
+
+from greynoise import GreyNoise
+
+from .constants import INTEGRATION_NAME, LOGGER_NAME
+
+logger = get_logger(LOGGER_NAME)
+
+
+def gnql_query(config, params):
+    query = params.get("query")
+    max_results = str(params.get("max_results"))
+    if not query:
+        raise ConnectorError("Missing required input")
+
+    if not max_results:
```

Comment:
can we define an upper limit to ensure it doesn't load too much of data into memory

Reply:
By default, it maxes out at 1,000 IPs in the response, which I think should be okay?  Or would you desire a lower limit than that?

Reply:
1000 IPs should be good